diff --git a/node_modules/@react-native/community-cli-plugin/dist/commands/start/runServer.js b/node_modules/@react-native/community-cli-plugin/dist/commands/start/runServer.js
--- a/node_modules/@react-native/community-cli-plugin/dist/commands/start/runServer.js
+++ b/node_modules/@react-native/community-cli-plugin/dist/commands/start/runServer.js
@@ -113,16 +113,19 @@
     middleware: communityMiddleware,
     websocketEndpoints: communityWebsocketEndpoints,
     messageSocketEndpoint,
     eventsSocketEndpoint,
   } = (0, _middleware.createDevServerMiddleware)({
     host: hostname,
     port,
     watchFolders,
   });
   const { middleware, websocketEndpoints } = (0,
   _devMiddleware.createDevMiddleware)({
     serverBaseUrl: devServerUrl,
     logger: (0, _createDevMiddlewareLogger.default)(terminalReporter),
+    unstable_experiments: {
+      enableStandaloneFuseboxShell: false,
+    },
   });
   metroConfig.reporter = {
     update(event) {
